src/Microsoft.AspNet.Razor/Generator/GeneratedTagHelperContext.cs

```diff
@@ -79,6 +80,21 @@ public GeneratedTagHelperContext()
         public string ExecutionContextOutputPropertyName { get; set; }
 
         /// <summary>
+        /// The name of the method used to format an error message about using an indexer when the tag helper property
+        /// is <c>null</c>.
+        /// </summary>
+        /// <remarks>
+        /// Method signature should be
+        /// <code>
+        /// public string FormatInvalidIndexerAssignment(
```

Comment:
Love the idea for this, we should file an issue to do this for all of the config properties